### PR TITLE
#157110843 add fitbit support 

### PR DESCRIPTION
**Pull request by mutoromike**
**What does this PR do?**

Allows a user to integrate Wger to work with Fitbit, which is a more advanced application that combines all nutritional factors such as workouts and exercises, nutrition etc. By adding Fitbit support, a user is able to see information while not having to log in to Fitbit every single time.

**Sync from Fitbit button that allows a onetime authentication**
screen shot 2018-05-16 at 08 31 03

**What are related PT stories?**
Add Fitbit support. 
ID: #157110843

### CODE REVIEW

**Review by meshnesh — CHANGES_REQUESTED** [review]
request for reviewers for the PR and fix the conflicting files

File: wger/core/views/fitbit.py [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+        url = os.environ.get('url')
+        response = requests.get(url, headers=headers)
+
+        if response.status_code == 200:
+            return response.json()
+        elif response.status_code == 401:
+            token = self.RefreshToken(token)
+            self.GetWeight(token)
+        else:
+            raise Exception("Action unsuccessful")
```
Comment (FelixWambiri) [inline wger/core/views/fitbit.py:112]: Nice Implementation especially good work on the comments.It makes it easy to go through and understand.

**Review by FelixWambiri — APPROVED** [review]
LGTM

**Review by lakivisi-zz — CHANGES_REQUESTED** [review]
Upload screenshot of the whole view with the button

File: .travis.yml [hunk 2]
```diff
@@ -59,10 +59,12 @@ script:
   - gulp lint
 
   # Regular application
-  - coverage run --source='wger' ./manage.py test
+  - coverage run --source='.' ./manage.py test
```
Comment (Kalela) [inline .travis.yml]: Good work keeping track of your old task throughout the development period

**Review by Kalela — APPROVED** [review]
LGTM

File: wger/core/views/misc.py [hunk 3]
```diff
@@ -209,3 +210,36 @@ def form_valid(self, form):
         mail.mail_admins(subject, message)
 
         return super(FeedbackClass, self).form_valid(form)
+
+
+@login_required
+def fitbitLogin(request):
```
Comment (frankip) [inline wger/core/views/misc.py:216]: 👍

**Review by frankip — APPROVED** [review]
LGTM

**Comment by mutoromike** [comment]
After syncing with Fitbit, the data is presented as shown below:
screen shot 2018-05-17 at 12 13 47

**Review by betsybeth — APPROVED** [review]
LGTM

**Review by meshnesh — APPROVED** [review]

**Review by lakivisi-zz — APPROVED** [review]

**Pull request merged**